makefile

```diff
@@ -253,6 +257,8 @@ help:
 	@echo "  TESTCASES - When set, 'test' target runs only the specified test cases. The"
 	@echo "      value is used for the -k option of pytest (see 'pytest --help')."
 	@echo "      Optional, defaults to running all tests."
+	@echo "  TESTSERVER - Optional: Nickname of test server or server group in WBEM server"
+	@echo "      definition file for end2end tests. Default: 'default'"
```

Comment:
Does this mean that it looks for a server or group named default???

Reply:
yes

Reply:
Comment below clarifys this if we show the default in the example.